cordapp/src/main/kotlin/net/corda/plugins/Utils.kt

```diff
@@ -13,8 +12,11 @@ import kotlin.math.max
  * Mimics the "project.ext" functionality in groovy which provides a direct
  * accessor to the "ext" extension (See: ExtraPropertiesExtension)
  */
-@Suppress("UNCHECKED_CAST")
-fun <T : Any> Project.ext(name: String): T = (extensions.findByName("ext") as ExtraPropertiesExtension).get(name) as T
+
+fun <T : Any> Project.findRootProperty(name: String): T? {
+    return rootProject.property(name) as T?
```

Comment:
Good, but this will throw an exception if the property cannot be found (see [here](https://docs.gradle.org/current/javadoc/org/gradle/api/Project.html#property-java.lang.String-)). If you want it to return `null` for absent properties - and I think you _do_ - then you'll need to use [findProperty](https://docs.gradle.org/current/javadoc/org/gradle/api/Project.html#findProperty-java.lang.String-) instead.